src/core/useCases/feed/feed.actions.ts

```diff
@@ -164,6 +165,13 @@ function reopenEntourageSucceeded(payload: { entourageUuid: string; }) {
   }
 }
 
+function toggleFeedFilter(payload: { type: FilterEntourageType; category?: FilterFeedCategory;}) {
```

Comment:
`setFeedFilter` me semble plus approprié, car c'est pas un simple toggle.

```suggestion
function setFeedFilter(payload: { type: FilterEntourageType; category?: FilterFeedCategory;}) {
```

Reply:
C'est vrai mais je trouve qu'on est encore plus loin qu'un set ^^. Vu que je maj à l'opposé de l'état précedent. Mais so ça vous semble plus claire à vous deux, go!

Reply:
Moi je suis plutôt pour le `toggle` également, vu qu'on l'active ou désactive avec la même action

Reply:
Quitte à faire 2 actions, une pour le `type` et une pour la `category`?